src/Resources/views/Download/creditMemo.html.twig

```diff
@@ -56,59 +66,69 @@
                                 {% endif %}
                             {% endif %}
                         </td>
-
-                        <td>
-                            <strong>{{ from.customerName }}</strong><br/>
-                            {% if from.company %}{{ from.company }}<br/>{% endif %}
-                            {{ from.street }}<br/>
-                            {{ from.city }}<br/>
-                            {% if from.provinceName %}{{ from.provinceName }}{% endif %}
-                            {{ from.countryCode|sylius_country_name|upper }} {{ from.postcode }}
-                        </td>
                     </tr>
                 </table>
             </td>
         </tr>
         {% endif %}
 
         <tr class="heading">
-            <td>{{ 'sylius.ui.item'|trans }}</td>
-            <td>{{ 'sylius.ui.tax_total'|trans }}</td>
-            <td>{{ 'sylius.ui.total'|trans }}</td>
+            <td>{{ 'sylius_refund.ui.no'|trans }}</td>
+            <td>{{ 'sylius.ui.name'|trans }}</td>
+            <td>{{ 'sylius.ui.quantity'|trans }}</td>
+            <td>{{ 'sylius_refund.ui.unit_net_price'|trans }}</td>
+            <td>{{ 'sylius_refund.ui.net_value'|trans }}</td>
+            <td>{{ 'sylius_refund.ui.tax_rate'|trans }}</td>
+            <td>{{ 'sylius_refund.ui.tax_amount'|trans }}</td>
+            <td>{{ 'sylius_refund.ui.gross_value'|trans }}</td>
+            <td>{{ 'sylius.ui.currency'|trans }}</td>
         </tr>
 
-        {% for unit in creditMemo.units %}
-        <tr class="item">
-            <td>{{ unit.productName }}</td>
-            <td>{{ unit.taxesTotal|sylius_format_money(creditMemo.currencyCode, creditMemo.localeCode) }}</td>
-            <td>{{ unit.total|sylius_format_money(creditMemo.currencyCode, creditMemo.localeCode) }}</td>
-        </tr>
+        {% for item in creditMemo.lineItems %}
+            <tr class="item">
+                <td>{{ loop.index }}</td>
+                <td>{{ item.name }}</td>
+                <td>{{ item.quantity }}</td>
+                <td>{{ (item.unitNetPrice/100)|number_format(2) }}</td>
```

Comment:
I'm not sure whether it'll render prices like `4.20` correctly - `number_format` might cut it to `4.2` if not for those parentheses.

```suggestion
                <td>{{ '%0.2f'|format(item.unitNetPrice/100) }}</td>
```

This will make sure there are always two digits to the right of the decimal point. I'm not too convinced to any option though.